content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -71,22 +71,22 @@ separate endpoint for each group version.
 
 ### Aggregated discovery
 
-{{< feature-state state="beta" for_k8s_version="v1.27" >}}
+{{< feature-state state="stable" for_k8s_version="v1.27" >}}
```

Comment:
Is this right? It actually graduated to stable in v1.27?

Otherwise, try:
```suggestion
{{< feature-state feature_gate_name="AggregatedDiscoveryEndpoint" >}}
```

(I guessed at the name of the associated feature gate; please check it)